integration/kubernetes/k8s-port-forward.bats

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+source /etc/os-release || source /usr/lib/os-release
+
+setup() {
+	[ "$ID" != "ubuntu" ] && skip "test will run only on ubuntu"
+
+	export KUBECONFIG="$HOME/.kube/config"
+	get_pod_config_dir
+
+	# Check redis-cli is installed
+	check_package=$(which redis-cli | wc -l)
+
+	# Install redis-cli if is not installed
+	if [ "$check_package" -eq 0 ]; then
+		sudo -E apt install -y redis-server
+	fi
+}
+
+@test "Port forwarding" {
+	[ "$ID" != "ubuntu" ] && skip "test will run only on ubuntu"
+        
+	deployment_name="redis-master"
+
+	# Create deployment
+	kubectl apply -f "${pod_config_dir}/redis-master-deployment.yaml"
+
+	# Check deployment
+	kubectl wait --for=condition=Available deployment/"$deployment_name"
+	kubectl expose deployment/"$deployment_name"
+
+	# Get pod name
+	pod_name=$(kubectl get pods --output=jsonpath={.items..metadata.name})
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	# View replicaset
+	kubectl get rs
+
+	# Create service
+	kubectl apply -f "${pod_config_dir}/redis-master-service.yaml"
+
+	# Check service
+	kubectl get svc | grep redis
+
+	# Check redis service
+	port_redis=$(kubectl get pods $pod_name --template='{{(index (index .spec.containers 0).ports 0).containerPort}}{{"\n"}}')
+
+	# Verify that redis is running in the pod and listening on port 6379
```

Comment:
I'd remove the port number from the comment. It's clear from the code what the number is so it's duplication.